addons/website_sale/views/variant_templates.xml

```diff
@@ -155,6 +156,37 @@
                             </li>
                         </ul>
                     </t>
+
+                    <t t-elif="attribute.display_type == 'image'">
+                        <ul
+                            t-att-data-attribute_id="attribute.id"
+                            t-attf-class="list-inline o_wsale_product_attribute #{'d-none' if single_and_custom else ''}"
+                        >
+                            <li t-foreach="ptavs" t-as="ptav" class="list-inline-item me-2 mb-2">
+                                <label
+                                    name="attribute_image_selector"
```

Comment:
nitpicking:
Let's try to make the selector more specific by adding the prefix `o_` to specify that this selector is used by Odoo, and `wsale_` to specify that it's an eCommerce feature. This will help understanding what is this selector, and avoid conflicting with potential partner modules.
```suggestion
                                    name="o_wsale_attribute_image_selector"
```